### PR TITLE
Better multiple language support

### PR DESCRIPTION
**Pull request by asottile**

### CODE REVIEW

**Comment by asottile** [comment]
This is blocked on this travis-ci regression: https://github.com/travis-ci/travis-ci/issues/3909

**Comment by struys** [comment]
lg2m

File: pre_commit/git.py [hunk 1]
```diff
@@ -37,9 +37,10 @@ def is_in_merge_conflict():
 def parse_merge_msg_for_conflicts(merge_msg):
     # Conflicted files start with tabs
     return [
```
Comment (struys) [inline pre_commit/git.py]: I assume this has nothing to do with the actual change.

Comment (asottile) [inline pre_commit/git.py]: yeah no, I just updated git on my home machine and couldn't get the tests passing without this.

**Comment by asottile** [comment]
This has been released as part of 0.5.0

**Pull request merged**